assets/src/edit-story/components/button/index.js

```diff
@@ -41,13 +41,14 @@ const Base = styled.button.attrs(({ isDisabled }) => ({
   border-style: solid;
   border-radius: 2px;
   background: transparent;
-  display: block;
+  display: inline-block;
   min-width: ${({ isIcon }) => (isIcon ? 'initial' : '63px')};
   line-height: 28px;
-  height: 30px;
+  max-height: 30px;
```

Comment:
From my testing, this doesn't break any existing button.
Maybe we should unify the buttons at some point (we have at least 2 that look similar).

Reply:
Definitely need to unify the buttons. `max-height` is tricky because text can sometimes wrap to th next line. Can we get away with not specifying `height`/`max-height` at all?

Agreed that we need to clean up buttons (and plenty of other things) at some point.

Reply:
I was testing that with long text in `Set as background` button, but that's another Button component...

I'm 99% sure it will not break anything if we remove the height, looks like it was introduced in the old editor? For that 1% I did backwards compatible version.

Reply:
It will break this (on really small window):
![image](https://user-images.githubusercontent.com/21036224/85136132-7b53e180-b23f-11ea-9bb9-02ae7bec8e06.png)

Reply:
Sounds like an edge case, although we also have to consider translations.

Can you create an issue for unifying button components (`Type: Code Quality`)?

Then we can leave this as-is for now here.

Thanks for testing 👍